camera_cards

```diff
@@ -236,9 +242,11 @@ if [[ "${AIP}" == "YES" ]] ; then
         done <"${TEMP_AUDIOLIST}"
         ffmpeg -f concat -safe 0 -i "${TEMP_CONCATLIST}" -i "${FIRST_FILE}" -map 0 -map_metadata 1 -c copy "${CONCATENATED_AUDIO_FILE}"
         _writelog "CONCATENATED AUDIO FILE" "${CONCATENATED_AUDIO_FILE}"
+        _writelog "Audio concatenation process finished" "$(date +%FT%T)"
```

Comment:
nbd, but maybe at some point, could at a `-t` option to `_writelog` to mean that the value of `key:value` should be a timestamp of now.

Reply:
👍🏻 859d8a3f2034021c316a069f7b2e763253cd4fce